src/test/java/com/booster/server/VocabularyEntryControllerTest.java

```diff
@@ -65,25 +73,61 @@ void returnVocabularyEntryListOfPredefinedSize() throws Exception {
 
         vocabularyEntryRepository.save(new VocabularyEntryEntity()
                 .setName(coalesce)
-                .setDescription(coalesceDescription));
+                .setDescription(coalesceDescription)
+                .setLastSeenAt(LocalDateTime.now()));
         vocabularyEntryRepository.save(new VocabularyEntryEntity()
                 .setName(robust)
-                .setDescription(robustDescription));
+                .setDescription(robustDescription)
+                .setLastSeenAt(LocalDateTime.of(LocalDate.EPOCH, LocalTime.MIN)));
 
         mockMvc.perform(MockMvcRequestBuilders.get("/vocabulary-entry/list?size=%s".formatted(expectedSize)))
                 .andDo(print())
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.length()").value(expectedSize))
-                .andExpect((jsonPath("$.[0].name").value(coalesce)))
-                .andExpect((jsonPath("$.[0].description").value(coalesceDescription)))
-                .andExpect((jsonPath("$.[1].name").value(robust)))
-                .andExpect((jsonPath("$.[1].description").value(robustDescription)));
-
+                .andExpect((jsonPath("$.[0].name").value(robust)))
+                .andExpect((jsonPath("$.[0].description").value(robustDescription)))
+                .andExpect((jsonPath("$.[1].name").value(coalesce)))
+                .andExpect((jsonPath("$.[1].description").value(coalesceDescription)));
 
         assertThat(vocabularyEntryRepository.findAll())
                 .hasSize(expectedSize)
                 .extracting("name", "description")
                 .containsExactly(Tuple.tuple(coalesce, coalesceDescription), Tuple.tuple(robust, robustDescription));
```

Comment:
This is redundant and can be deleted as the `GET` request above already tests that.
The same is true for all vocabulary entry/note tests.